AppCenterDemoApp/Assets/Puppet/PuppetTransmission.cs

```diff
@@ -446,9 +450,17 @@ public void TrackEventStringPropertiesChildTransmission()
             else
             {
                 var propertyConfigurator = transmissionTarget.GetPropertyConfigurator();
-                foreach (var property in properties)
-                {
-                    propertyConfigurator.SetEventProperty(property.Key, property.Value);
+                if (properties != null) {
+                  foreach (var property in properties)
```

Comment:
Please fix indentation (it should be 4 spaces)

Reply:
Update it in new commit.